assets/style.css

```
@@ -73,6 +73,47 @@
+
+.prose .collapsible > .collapsible-title:empty {
+    display: none;
+}
+
+.prose details.collapsible-item {
+    @apply m-0 border-t-2 border-white dark:border-gray-800;
+}
+
+.prose details.collapsible-item > summary {
+    @apply flex items-center gap-3 px-4 py-3 bg-gray-100 dark:bg-gray-700 hover:bg-gray-200 dark:hover:bg-gray-600 transition-colors cursor-pointer select-none font-bold text-lg leading-snug text-black dark:text-white;
+    list-style: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)